modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -137,34 +222,227 @@ export default function EditObjectFolder({
 				/>
 			)}
 
-			{showModal.editObjectFolder && (
+			{modelBuilderModals.addObjectField && (
+				<ModalAddObjectField
+					baseResourceURL={baseResourceURL}
+					creationLanguageId={
+						selectedObjectDefinitionNode?.data
+							?.defaultLanguageId as Liferay.Language.Locale
+					}
+					objectDefinitionExternalReferenceCode={
+						selectedObjectDefinitionNodeERC
+					}
+					objectDefinitionName={
+						selectedObjectDefinitionNode?.data?.name as string
+					}
+					onAfterSubmit={(newObjectField) => {
+						const {edges, nodes} = store.getState();
```

Comment:
Hey @Artartur, the `edges` and `nodes` has already be created in global scope of this file. So you can remove this repeated declaration here.

Reply:
Done!